ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Pipelines.resource

```diff
@@ -34,6 +35,31 @@ Create Pipeline Server
     Wait Until Generic Modal Disappears
     Wait Until Project Is Open    project_title=${project_title}
     ...    timeout-pre-spinner=5s    timeout-spinner=60s
+    # Check if there is any CA Bundle
+    # I did not find a better way to check if multi-line string is empty or "NONE"
+    ${length}=    Get Length    "${aws_s3_ca_bundle}"
+    IF    ${length} <= 6
+        ${aws_s3_ca_bundle}=    Get Variable Value    ${S3.AWS_CA_BUNDLE}    ${NONE}
```

Comment:
I think the keyword should not set the value from a global variable here if not set as a parameter.

I think keywords calling `Create Pipeline Server` should pass it as a parameter if the need ${S3.AWS_CA_BUNDLE} 

Another option could be to set the default value like this:
```
Create Pipeline Server
    [Documentation]    Creates the DS Pipeline server from DS Project details page
    ...                It assumes the Data Connection is aleady created
    ...                and you wants to use defaul DB configurations [TEMPORARY]
    [Arguments]    ${dc_name}    ${project_title}
    ...            ${s3_ca_bundle}=${S3.AWS_CA_BUNDLE} 
 ```

Reply:
The reason for doing this is to not force setting the ca bundle as a parameter (but it is allowed).

Moreover, this other suggestion will not work
```
    [Arguments]    ${dc_name}    ${project_title}
    ...            ${s3_ca_bundle}=${S3.AWS_CA_BUNDLE}
```
because `S3.AWS_CA_BUNDLE` is optional. If it is not set in test-variables then you will get error.

I did not want to make it as a mandatory field in test-variables as it could introduce other issues
* could break local devel env if people are not aware of adding a new mandatory field
* confusions what the empty CA bundle means (is that simple http? is that public url like aws? ...)

Reply:
Hi @jgarciao is such an explanation ok?

Reply:
As we discussed, this code will be soon simplified when RHOAIENG-1926 is done so ok for me to merge it like this